lib/oli_web/live/admin/institutions/index_live.ex

```diff
@@ -0,0 +1,617 @@
+defmodule OliWeb.Admin.Institutions.IndexLive do
+  use OliWeb, :live_view
+  require Logger
+
+  alias Oli.Institutions
+  alias Oli.Predefined
+  alias Oli.Slack
+
+  alias OliWeb.Common.Breadcrumb
+  alias OliWeb.Router.Helpers, as: Routes
+  alias OliWeb.Components.Modal
+
+  alias Phoenix.LiveView.JS
+
+  on_mount(OliWeb.LiveSessionPlugs.SetCurrentAuthor)
+
+  def mount(_params, session, socket) do
+    institutions = Institutions.list_institutions()
```

Comment:
~NIT, we could directly assign its value in `assign` since we do not use the `institutions` value in another place.~
 I can tell we also need its value for institutions_list 😄